lib/controllers/widget/hardware_os_card.dart

```diff
@@ -0,0 +1,408 @@
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Plataform Version',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.platformVersion,
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Device ID',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.deviceId,
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Last Enrollment Time',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.lastEnrollmentTime,
```

Comment:
Except for like 3-5 properties everything is optional, so can you please wrap all of the properties around the function getValueOrDefault

Reply:
All the ones for which it applies, because for example in auto update expiration it doesn't apply.

Reply:
Done.